docs/CompatibleSoftware.rst

```diff
@@ -87,96 +215,118 @@ color processing. This profile is useful for those who want to mirror the native
-photoshop.  The core idea is to create an .icc profile, with a valid
-description, and then to save it to the proper OS icc directory. (On
-OSX, ``~/Library/ColorSync/Profiles/``). Upon a Photoshop relaunch,
-Edit->Assign Profile, and then select your new OCIO lut.
+OpenColorIO display luts can be exported as ICC profiles for use in photoshop. The core idea is to create an .icc profile, with a valid description, and then to save it to the proper OS icc directory. (On OSX, ``~/Library/ColorSync/Profiles/``). Upon a Photoshop relaunch, Edit->Assign Profile, and then select your new OCIO lut.
 
-See the the OCIO user guide `for details on baking ICC profiles for Photoshop
-<userguide-bakelut-photoshop>`__
+Website : `<https://www.adobe.com/products/photoshop.html?promoid=PC1PQQ5T&mv=other>`__
 
-OpenImageIO
-***********
 
-`Open Source Image Library / Renderer Texture Engine <http://openimageio.org>`__
+Python
+******
+
+The OpenColorIO API is available for use in python. See the "pyglue" directory
+in the codebase.
 
-Available in the current code trunk. Integration is with makecolortx (allowing
-for color space conversion during mipmap generation), and also through the
-public header `src/include/color.h <http://github.com/OpenImageIO/oiio/blob/master/src/include/color.h>`__ .
+Documentation :
 
-Remaining integration tasks include
-`color conversion at runtime <http://github.com/OpenImageIO/oiio/issues/193>`__ .
+- See the developer guide for `usage examples <_developers-usageexamples>`__ and API documentation on the Python bindings.
 
-C++
-***
 
-The core OpenColorIO API is available for use in C++. See the `export
-directory
-<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/export/OpenColorIO>`__
-for the C++ API headers.  Minimal code examples are also available in
-the source code distribution. Of particular note are
-`src/apps/ocioconvert/
-<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/apps/ocioconvert>`__
-and `src/apps/ociodisplay/
-<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/apps/ociodisplay>`__
+RV (Autodesk)
+*************
 
-Also see the :ref:`developer-guide`
+Website : `<http://www.tweaksoftware.com>`__
 
-Python
-******
+Supported Version:  >= 4
 
-The OpenColorIO API is available for use in python. See the "pyglue" directory
-in the codebase.
+Documentation : 
 
-See the developer guide for `usage examples
-<developers-usageexamples>`__ and API documentation on the PYthon
-bindings
+- For more details, see the OpenColorIO section of the `RV User Manual <http://www.tweaksoftware.com/static/documentation/rv/current/html/rv_manual.html#OpenColorIO>`__.
 
-Vegas Pro
-*********
 
-`Video editing - Sony <http://www.sonycreativesoftware.com/vegaspro>`__
+Silhouette (Boris FX)
+*********************
+
+Website : `<http://www.silhouettefx.com/silhouette>`__
+
+Supported Version: >= 4.5
+
+Documentation : 
```

Comment:
This link is dead. Let's delete this Documentation section and instead give Silhouette a description:

```
OCIO is natively integrated in Silhouette. Full support is provided for both image import/export, as well as image display.
```